### PR TITLE
docs: add link to discovery service in kubespan

### PR DESCRIPTION
**Pull request by TimJones**
Add more contextual description of the discovery service
to the KubeSpan docs, and a link to the repo.

Signed-off-by: Tim Jones <tim.jones@siderolabs.com>

---
This change is [Reviewable](https://reviewable.io/reviews/siderolabs/talos/5727)

### CODE REVIEW

**Comment by steverfrancis** [comment]
It actually needs to say “download” on the link, to meet silly trademark requirements…

**Comment by TimJones** [comment]
@steverfrancis ah I see :/ 

Is "can be <downloaded here>." OK? Or does it really need to be completely separate hard "Download here" on-its-own link?

File: website/content/v1.0/kubernetes-guides/network/kubespan.md [hunk 1]
```diff
@@ -8,6 +8,9 @@ aliases:
 KubeSpan is a feature of Talos that automates the setup and maintenance of a full mesh [WireGuard](https://www.wireguard.com) network for your cluster, giving you the ability to operate hybrid Kubernetes clusters that can span the edge, datacenter, and cloud.
 Management of keys and discovery of peers can be completely automated for a zero-touch experience that makes it simple and easy to create hybrid clusters.
 
+KubeSpan consists of client code in Talos Linux, as well as a discovery service that enables clients to securely find each other.
+Sidero Labs operates a free Discovery Service, but the discovery service may be operated by your organization and can be [downloaded here](https://github.com/siderolabs/discovery-service).
```
Comment (smira) [inline website/content/v1.0/kubernetes-guides/network/kubespan.md:12]: should we note that one can't run simply download and run it, as it's BSL?

Comment (steverfrancis) [reply website/content/v1.0/kubernetes-guides/network/kubespan.md:12]: "downloaded here" is fine. We should document how to setup/run it *somewhere*, but that is a separate issue...

**Review by smira — APPROVED** [review]

**Pull request merged**